RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -1033,6 +1033,18 @@ boolean auto_post_adventure()
 			abort("We have been disavowed...");
 		}
 	}
+	
+	if(isActuallyEd() && my_hp() == 0)
+	{
+		acquireHP(1);
+		if(my_hp() == 0 && get_property("_auto_edNoLinenCheck").to_boolean())	//we got beaten up in a risky NC as ed
+		{
+			if(!edUnderworldAdv())	//try to spend 1 adv to restore HP
+			{
+				abort("Failed to rest as ed");
+			}
+		}
+	}
```

Comment:
We already have a call to `acquireHP(1)` for Ed in auto_pre_adv. Why is this necessary here rather than there?

Reply:
I put it in post adv to catch _auto_edNoLinenCheck before it is reset, as well as to preempt our counter incrementing for dying in combat.
Thus making sure we are at 0 HP because of a noncombat that killed us rather than a combat loss.
The latter are being tracked separately to make sure we are not dying too many times

Maybe I should rework this.

Reply:
change made